app/views/smart_proxies/plugins/_dhcp_subnets.html.erb

```diff
@@ -0,0 +1,66 @@
+<ul id="proxy-dhcp-tab" class="nav nav-tabs nav-tabs-pf nav-stacked col-md-3">
+  <li class="active"><a href="#dhcp_general"><%= _('General') %></a></li>
+  <% subnets.each do |subnet| %>
+    <li><a href="#<%= subnet.network.parameterize %>"
+           data-placeholder="<%= subnet.network.parameterize %>"
+           data-netmask="<%= subnet.netmask %>"
+           class="subnet-menu"
+           data-url="<%= details_smart_proxy_proxy_subnets_path(@smart_proxy)%>"><%= subnet_label subnet, @smart_proxy %></a></li>
+  <% end %>
+</ul>
+<div id="proxy-dhcp-tab-content" class="tab-content col-md-9">
+  <div class="tab-pane active in" id="dhcp_general">
+    <div class="container-fluid container-cards-pf">
+      <%= show_feature_version('dhcp') %>
+    </div>
+
+
+    <div class="container-fluid container-cards-pf">
+      <% subnets.each do |subnet| %>
+        <div class="row row-cards-pf">
+          <% if net = foreman_subnet(subnet, @smart_proxy) %>
```

Comment:
Why split this into two cards per subnet? Host & domain count can be displayed in one card.